spec/timeliness/definitions_spec.rb

```diff
@@ -113,4 +113,17 @@
       threads.each { |t| expect(t.value).to eql(Time.new(2016,06,30)) }
     end
   end
+
+  context "add_format_tokens" do
+    it "should allow add custom format tokens" do
+      expect(parser._parse('01/02/2000', :date, format: '%d/%m/%Y')).to be_nil
+      definitions.add_format_tokens({
```

Comment:
The other examples don't have a newline between `expect` and `definitions`. 
Yours doesn't have one between lines 119 and 120, but it does have one on line 125. 
I think either we can either insert a newline between lines 119 and 120 to be consistent, or remove the newline from line 125 to be consistent.